htdocs/societe/class/api_contacts.class.php

```diff
@@ -465,7 +465,7 @@ public function addCategory($id, $category_id)
 		if (!DolibarrApi::_checkAccessToResource('contact', $this->contact->id)) {
 			throw new RestException(401, 'Access not allowed for login '.DolibarrApiAccess::$user->login);
 		}
-		if (!DolibarrApi::_checkAccessToResource('category', $category->id)) {
+		if (!DolibarrApi::_checkAccessToResource('categorie', $category->id)) {
```

Comment:
Because future and followikg version are using "category", we must keep to use this key zs it is the target to switch to english keys.
Instead we must backport the same way to fix that done into recent version, so adding a compatibility test inside the function.